src/main/webapp/get-started.html

```diff
@@ -0,0 +1,308 @@
+                  https://karaf.apache.org/download.html</a>.</li>
+                <li>Download the binary distribution of Karaf Runtime that matches your system (zip for windows, tar.gz for unixes).</li>
+                <li>Extract the archive to a new folder on your hard drive. For example in <code>/opt/karaf</code>, from now on this directory will be
+                  referenced as <code>$KARAF_HOME</code>.</li>
+              </ol>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-2">
+          <div class="col">
+            <h4 class="pb-3 mb-1">Start and connect</h4>
+            <ol>
+              <li>Open a command line console and change the directory to <code>$KARAF_HOME</code>.</li>
+              <li>To start the server, run the following command on Unix:
+                <pre class="alert alert-primary"><code>$KARAF_HOME/bin/karaf</code></pre>
+                Respectively on Windows:
+                <pre class="alert alert-primary"><code>$KARAF_HOME\bin\karaf.bat</code></pre>
+              </li>
+              <li>You are now connected to the Karaf shell!</li>
+            </ol>
+
+            <p>
+              <pre class="alert alert-primary"><code>
+        __ __                  ____
+       / //_/____ __________ _/ __/
+      / ,<  / __ `/ ___/ __ `/ /_
+     / /| |/ /_/ / /  / /_/ / __/
+    /_/ |_|\__,_/_/   \__,_/_/
+
+  Apache Karaf (4.2.0)
+
+Hit '<tab>' for a list of available commands
+and '[cmd] --help' for help on a specific command.
+Hit '<ctrl-d>' or type 'system:shutdown' or 'logout' to shutdown Karaf.
+
+karaf@root()>
+              </code></pre>
+            </p>
+            <p><i class="fa fa-info-circle"></i> If you want to launch your Karaf instance in background, run:
+              <pre class="alert alert-primary"><code>$KARAF_HOME/bin/start</code></pre>
+            </p>
+          </div>
+        </div>
+
+        <!-- Easy to manage -->
+        <div class="row mb-2 mt-3">
+          <div class="col">
+            <h2 class="pb-3 mb-4 font-italic border-bottom"><i class="fas fa-laptop"></i> Easy to manage</h2>
+            <p>You can manage your Karaf instance with the shell console to a local instance or to a remote instance using the ssh client.</p>
+          </div>
+        </div>
+
+        <div class="row mb-2 mt-2">
+          <div class="col">
+            <h4 class="pb-3 mb-1">Connect to the shell console</h4>
+            <ol>
+              <li>Open a command line console and change the directory to <code>$KARAF_HOME</code>.</li>
+              <li>To connect to the local instance, run:
+                <pre class="alert alert-primary"><code>$KARAF_HOME/bin/client</code></pre></li>
+            </ol>
+            <p><i class="fa fa-info-circle"></i> To connect to a remote instance, run:</p>
+            <pre class="alert alert-primary"><code>$KARAF_HOME/bin/client -a "IP" -p "PORT"</code></pre>
```

Comment:
Maybe mention regular `ssh` as well, and note the `client` is a multi-platform `ssh` client.

Reply:
changes done!